src/components/config/ecommerce/vtex/Config.vue

```diff
@@ -139,18 +156,42 @@
         'errorConnectVtexCatalog',
         'sellersList',
         'errorSellersList',
+        'errorSyncSellers',
+        'checkSellers',
       ]),
       sellerOptions() {
-        return this.sellersList || [];
+        return (
+          this.sellersList.map((item) => {
+            return {
+              value: item,
+              label: item,
+            };
+          }) || []
```

Comment:
You can improve this "unnecessary" return code to this:
```suggestion
          this.sellersList.map((item) => ({
              value: item,
              label: item,
            })) || []
```